discord/state.py

```diff
@@ -783,6 +783,17 @@ def parse_message_reaction_remove(self, data) -> None:
         emoji_id = utils._get_as_snowflake(emoji, "id")
         emoji = PartialEmoji.with_state(self, id=emoji_id, name=emoji["name"])
         raw = RawReactionActionEvent(data, emoji, "REACTION_REMOVE")
+
+        member_data = data.get("member")
+        if member_data:
+            guild = self._get_guild(raw.guild_id)
+            if guild is not None:
+                raw.member = Member(data=member_data, guild=guild, state=self)
+            else:
+                raw.member = None
+        else:
+            raw.member = None
+
```

Comment:
This should be in the event class, not here

Reply:
While I would normally agree, the code I added was taken from the `parse_message_reaction_add` function, directly above: https://github.com/jordan-day/pycord/blob/83e1bebd2c363e6dd66c45900d75a4ffde24f5f1/discord/state.py#L750

Reply:
I can see why that was done now, thanks